examples/zephyr_example/boards/esp32_devkitc_wrover.overlay

```diff
@@ -1,34 +0,0 @@
-/*
```

Comment:
Could you explain the reasoning behind removing the board dt overlay? I'm a bit confused on this part.

Reply:
Sometimes there are too many boards overlay that are copy and paste stuff. For those cases, there is the `socs` folder where you can put all `overlays/conf` common on the SoC level. This was the exact case.